app/helpers/application_helper.rb

```diff
@@ -1,2 +1,13 @@
 module ApplicationHelper
+  def readable_date(date)
+   date.strftime("%B %e, %Y - %I:%M")
```

Comment:
It's good practice to leave a comment showing what the string will look like after formatting, to help out other devs / your future self.